lib/puppet_metadata/operatingsystem.rb

```diff
@@ -13,10 +13,10 @@ class OperatingSystem
     # The EOL dates for the various operating systems
     # @see .eol_date
     EOL_DATES = {
-      # https://almalinux.org/ FAQ at the bottom
+      # https://wiki.almalinux.org/release-notes/
       'AlmaLinux' => {
-        '9' => nil,
-        '8' => '2029-05-31',
+        '9' => '2027-05-31',
```

Comment:
It will remain supported:
> Version 9 will have active support until 31 May 2027, and security support until 31 May 2032.
```suggestion
        '9' => '2032-05-31',
```